backend/src/graphql/resolver/UserResolver.ts

```diff
@@ -493,13 +465,11 @@ export class UserResolver {
     try {
       // Save user
       await queryRunner.manager.save(user).catch((error) => {
-        logger.error('error saving user: ' + error)
-        throw new Error('error saving user: ' + error)
+        throw new LogError(`Error saving user: ${error}`, error)
```

Comment:
Too much information exposed? Do not expose the error
`throw new LogError('Error saving user', error)`